libs/sdk-backend-spi/src/common/paging.ts

```diff
@@ -17,4 +17,11 @@ export interface IPagedResource<TItem> {
      * @returns promise of a paged resource with the results of next page
      */
     next(): Promise<IPagedResource<TItem>>;
+
+    /**
+     * Request specific page of the resource
+     *
+     * @returns promise of a paged resource with the results of selected page
+     */
+    skip?(pageIndex: number): Promise<IPagedResource<TItem>>;
```

Comment:
Should we make this mandatory? Because it would be easier to use for the end user not having to check if skip is available.

Reply:
Also please document the API more: is the index zero-based for example?